tests/ui/.eslintrc.js

```diff
@@ -0,0 +1,85 @@
+module.exports = {
```

Comment:
Given this pattern will be repeated across other theme packages, it might make sense to move this into our lint config package